NodeJS/src/RestClient.ts

```diff
@@ -26,10 +26,16 @@ import { SkillCreateRequest, Subscription } from './models'
 export const STATION_GROUP_ID_NOT_EXISTS = -1
 
 const axios = require('axios').default
+
 const axiosCookieJarSupport = require('@3846masa/axios-cookiejar-support')
   .default
 const logger = log4js.getLogger('RestClient')
 
+axios.interceptors.request.use((request: any) => {
+  logger.debug('Starting Request', JSON.stringify(request, null, 2))
+  return request
+})
```

Comment:
is this interrupting ALL api calls? or am I misunderstanding?

Reply:
All api calls.

Reply:
cool, I was misunderstanding how that worked, thanks for clarifying!